packages/utils/lib/environment/detection.ts

```diff
@@ -22,3 +22,4 @@ export const flagHasScripts = isLocal || isEnterprise || isCloud || isTest;
 export const flagHasAuth = process.env['FLAG_AUTH_ENABLED'] !== 'false';
 export const flagHasManagedAuth =
     process.env['FLAG_MANAGED_AUTH_ENABLED'] === 'true' && Boolean(process.env['WORKOS_API_KEY'] && process.env['WORKOS_CLIENT_ID']);
+export const flagHasAPIRateLimit = process.env['FLAG_API_RATE_LIMIT_ENABLED'] !== 'false';
```

Comment:
I personally prefer disabling flags so without the right config the feature is enabled and must be disabled for non special scenario. The idea is to make it harder to misconfigure a service. (ie: more annoying from a devEx point of view but less prone to error in prod) What do you think?

Reply:
that's what it is, if not specified it's enabled hence the `!== 'false'`

 I'm just opting for a positive flag name instead of a negative one to avoid a double negative when disable is disabled, e.g: 

```
# harder to understand imo
FLAG_API_RATE_LIMIT_DISABLED=true
FLAG_API_RATE_LIMIT_DISABLED=false

# vs
FLAG_API_RATE_LIMIT_ENABLED=true
FLAG_API_RATE_LIMIT_ENABLED=false
```

wdyt?